test/Swashbuckle.AspNetCore.IntegrationTests/SwaggerVerifyIntegrationTest.cs

```diff
@@ -40,30 +40,23 @@ public async Task SwaggerEndpoint_ReturnsValidSwaggerJson(
         }
 
 #if NET6_0
-        [Fact]
-        public async Task SwaggerEndpoint_ReturnsValidSwaggerJson_Basic_DotNet_6()
+        [Theory]
+        [InlineData(typeof(Basic.Startup), "/swagger/v1/swagger.json")]
+        [InlineData(typeof(NSwagClientExample.Startup), "/swagger/v1/swagger.json")]
+        public async Task SwaggerEndpoint_ReturnsValidSwaggerJson_DotNet6(
+            Type startupType,
+            string swaggerRequestUri)
         {
             var testSite = new TestSite(typeof(Basic.Startup));
             using var client = testSite.BuildClient();
 
             using var swaggerResponse = await client.GetAsync("/swagger/v1/swagger.json");
             var swagger = await swaggerResponse.Content.ReadAsStringAsync();
-            await Verifier.VerifyJson(swagger);
+            await Verifier.Verify(swagger).UseParameters(startupType, GetVersion(swaggerRequestUri));
```

Comment:
Why change this from `VerifyJson`?

Reply:
the `SwaggerEndpoint_ReturnsValidSwaggerJson` uses `Verify`. I guess there is not much difference between them and `Verify` is more strict as it compares byte-by-byte. We might change to `VerifyJson` there, but I guess it's better to change it to `VerifyJson` in the main test as well (`SwaggerEndpoint_ReturnsValidSwaggerJson`) for consistency.

Reply:
For consistency and best experience with Verify we should consistently use `VerifyJson` - using just `Verify` was probably an oversight.

Reply:
Ah OK, I didn't realise that.